docker/install/ubuntu_install_vitis_ai_packages_ci.sh

```diff
@@ -23,7 +23,7 @@ set -o pipefail
 export PYXIR_HOME=/opt/pyxir
 mkdir "$PYXIR_HOME"
 
-pip3 install progressbar
+pip3 install progressbar h5py==2.10.0
```

Comment:
This probably requires your tutorial to be updated as well:
https://github.com/apache/tvm/blob/1455536cc22465cd6b7f6dde4f235c89ed4c5c3c/docs/deploy/vitis_ai.rst#L353

Note: this dependency is also mentioned on the [tensorflow installation script](https://github.com/apache/tvm/blob/0dc7de512b0328d15e85ba5c49234e9993493d2d/docker/install/ubuntu_install_tensorflow.sh) as `"h5py<3.0"`, so despite the duplication I think it is OK (quick mention to @areusch in this initiative to unify python deps).

Reply:
Yeah, it actually works with both 2.8.0 and 2.10.0 but I noticed a dependency issue with 2.8.0 (Tensorflow requires 2.10.0) and fixed it to 2.10.0, which works for us and should work for Tensorflow.

Reply:
This is the issue with 2.8.0:
`ERROR: tensorflow 2.3.1 has requirement h5py<2.11.0,>=2.10.0, but you'll have h5py 2.8.0 which is incompatible.`